packages/components/src/styles/components/form/super-select.scss

```
@@ -0,0 +1,490 @@
+      border-radius: var(--token-form-checkbox-border-radius);
+      cursor: pointer;
+      content: "";
+    }
+
+    // unchecked checkbox:
+    &:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-default);
+        border-color: var(--token-form-control-base-border-color-default);
+        box-shadow: var(--token-elevation-inset-box-shadow);
+      }
+    }
+
+    // unchecked checkbox, hovered:
+    &[aria-current="true"],
+    &:hover:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-hover);
+        border-color: var(--token-form-control-base-border-color-hover);
+      }
+    }
+
+    // unchecked checkbox, disabled:
+    // NOTE: may not be useful for this component
+    &[aria-disabled="true"]:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-disabled-surface-color);
+        border-color: var(--token-form-control-disabled-border-color);
+        box-shadow: none;
+        cursor: not-allowed;
+      }
+    }
+
+    // checked checkbox:
+    &[aria-selected="true"] {
+      &::before {
+        background-color: var(--token-form-control-checked-surface-color-default);
+        background-image: var(--token-form-checkbox-background-image-data-url);
+        border-color: var(--token-form-control-checked-border-color-default);
+      }
+
+      // checked checkbox, hovered:
+      &[aria-current="true"],
+      &:hover[aria-selected="true"] {
+        &::before {
+          background-color: var(--token-form-control-checked-border-color-default);
+          border-color: var(--token-form-control-checked-border-color-hover);
+        }
+      }
+
+      // checked checkbox, disabled:
+      // NOTE: may not be useful for this component
+      &[aria-disabled="true"][aria-selected="true"] {
+        &::before {
+          background-color: var(--token-form-control-disabled-surface-color);
+          background-image: var(--token-form-checkbox-background-image-data-url-disabled);
+          border-color: var(--token-form-control-disabled-border-color);
+          box-shadow: none;
+          cursor: not-allowed;
+        }
+      }
+    }
+  }
+}
+
+// NO MATCHES MESSAGE
+
+.hds-form-super-select .ember-basic-dropdown {
+  .ember-power-select-option--no-matches-message {
+    padding-left: 12px;
+    cursor: inherit;
+  }
+}
+
+// AFTER OPTIONS
+
+.hds-form-super-select__after-options {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  align-items: center;
+  padding: 8px 12px;
+  color: var(--token-color-foreground-strong);
+  font-size: var(--token-typography-body-100-font-size);
+  background-color: var(--token-color-surface-strong);
+  border-top: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+}
+
+.hds-form-super-select__result-count {
+  flex-grow: 1;
+  text-align: end;
```

could this do the trick?
```suggestion
  margin-left: auto;
```